### PR TITLE
News: GRASS GIS 8.3.2 released

### PR DESCRIPTION
**Pull request by neteler**
News item for the GRASS GIS 8.3.2 release.

:mega: Please someone donate a screenshot! :mega: 

Update of the relevant files:

- [x] data/grass.json
- [x] content/about/history/releases.md
- [x] content/news/2024_03_07_GRASS_GIS_8_3_2_released.md
- [x] static/images/news/grass832_TODO_TODO.png

### CODE REVIEW

File: content/news/2024_03_07_GRASS_GIS_8_3_2_released.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+### What's new in a nutshell
+
+The GRASS GIS 8.3.2 maintenance release contains more than **30 changes**
+compared to 8.3.1. This new patch release includes
+**important fixes and improvements** to the GRASS GIS modules and the
+graphical user interface (GUI), making it even more stable for daily work.
+
+Some of the most important changes are: fixes for `r.horizon` and some
+other modules; also database connections in vector data have been improved
+as well as selected manual pages.
```
Comment (veroandreo) [inline content/news/2024_03_07_GRASS_GIS_8_3_2_released.md:17]:
```suggestion
Some of the most important changes are: fixes for `r.horizon` and 
other modules; improvements in database connections for vector data 
as well as selected manual pages.
```

File: content/news/2024_03_07_GRASS_GIS_8_3_2_released.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+Translations continue in
+[Weblate](https://weblate.osgeo.org/projects/grass-gis/), which
+automatically creates pull requests with the translated chunks.
+We'd like to **thank the translators of all languages** for their
+ongoing support!
+
+<a href="/images/news/grass832_TODO.png">
+  <img src="/images/news/grass832_TODO.png"
```
Comment (veroandreo) [inline content/news/2024_03_07_GRASS_GIS_8_3_2_released.md:26]: What about this one that @cwhite911 made for a workshop last saturday? 
https://github.com/tomorrownow/intro-to-geoprocessing-workshop/blob/main/images/lc_spatial_scale.png

**Review by veroandreo — APPROVED** [review]

File: content/news/2024_03_07_GRASS_GIS_8_3_2_released.md (hunk 2 above)
Comment (neteler) [reply content/news/2024_03_07_GRASS_GIS_8_3_2_released.md]: (1a09f8f)

**Comment by neteler** [comment]
Seems the graphic didn't make it into the posting:
https://grass.osgeo.org/news/2024_03_07_grass_gis_8_3_2_released/

Confusing!

**Comment by neteler** [comment]
> Seems the graphic didn't make it into the posting: https://grass.osgeo.org/news/2024_03_07_grass_gis_8_3_2_released/
> 
> Confusing!

Ha, solved. There was a stray edit still in the `config.toml` on the server from the hugo 0.100 experiments.

So: `F5` and screenshot is now there as expected.

**Pull request merged**